meteor/client/ui/DragDropItemTypes.ts

```diff
@@ -1,4 +1,5 @@
 export enum DragDropItemTypes {
```

Comment:
The enum should be in singular form instead of plural.

Reply:
Now singular!